### ISSUE TITLE
Cannot reproduce the results provided in the "experiments" folder?

### ISSUE THREAD

**Issue by jiangxiluning** [ando-khachatryan/HiDDeN#33]
I created a training set by sampling 10000 images from the Coco datasets. I have some questions:

1. I can reproduce the "crop" experiment's training footprint with a similar scale of loss. But I cannot reproduce the 'combined' experiment. The total loss increased after several epochs and the training bitwise-error remained very high (0.46). It seems the model can not converge very well.
2. In the paper, the authors use YUV color space. Did you use YUV color space?

**Comment by ZYH-cu** [comment]
Hi,
do you reproduce the expected results now?
I have the same problem and the loss of the discriminator is abnormal..

**Comment by lei-account** [comment]
same question with about 0.4 bits error while combined
